Resources/config/services.yml

```diff
@@ -4,22 +4,22 @@ parameters:
     riper_security_active_directory_token_faulty : Riper\Security\ActiveDirectoryBundle\Security\Token\FaultyToken
 
     riper_security_tokens_classes :
-        standard : %riper_security_active_directory_token%
-        faulty : %riper_security_active_directory_token_faulty%
+        standard : "%riper_security_active_directory_token%"
+        faulty : "%riper_security_active_directory_token_faulty%"
 
 services:
     riper.security.active.directory.user.provider:
-        class: "Riper\Security\ActiveDirectoryBundle\Security\User\AdUserProvider"
+        class: Riper\Security\ActiveDirectoryBundle\Security\User\AdUserProvider
         arguments: ["%riper.security.active_directory.settings%", "@riper.security.active.directory.service.adldap", "@translator"]
 
     riper.security.active.directory.authentication.provider:
-        class: "Riper\Security\ActiveDirectoryBundle\Security\Authentication\AdAuthProvider"
+        class: Riper\Security\ActiveDirectoryBundle\Security\Authentication\AdAuthProvider
         arguments: ["@riper.security.active.directory.user.provider", "", "@riper.security.active.directory.service.adldap", "@translator", "%riper_security_tokens_classes%", %riper.security.active_directory.settings% ]
```

Comment:
Thanks for your PR, Deamon!
Just a short note: I think you have missed to quote the parameter %riper.security.active_directory.settings% here.